mitodl/ol-infrastructure
# enhancement: upgrade ocw hugo to 0.112.7

# What are the relevant tickets?
Relates to https://github.com/mitodl/ocw-studio/issues/1821 and https://github.com/mitodl/ol-infrastructure/pull/1532

# Description (What does it do?)
Upgrades Hugo in the ocw-course-publisher image from 0.112.0 to 0.112.7 (latest).

0.112.0 had a bug that was a problem for us, it is fixed in 0.112.7.

# How can this be tested?
1. `cd mitodl/ol-infrastructure/dockerfiles/ocw/node-hugo`
2. Build the image with `docker build . -t test-hugo`
3. Run the image with `docker run -it test-hugo bash`
4. Then verify the version with
    ```bash
    hugo version
    ```
